pkg/pillar/zedcloud/authen.go

```diff
@@ -384,68 +385,92 @@ func VerifySigningCertChain(ctx *ZedCloudContext, content []byte) ([]byte, error
 	err := proto.Unmarshal(content, sm)
 	if err != nil {
 		log.Errorf("SaveCertsToFile: Unmarshal err %v\n", err)
```

Comment:
Not your change, but the function name doesn't match the function.

Reply:
sure, done.